app/components/impersonation-menu.hbs

```diff
@@ -1,7 +1,7 @@
 <div class="au-u-flex au-u-flex--vertical-center">
   <AuPill @icon="user">{{this.adminLabel}}</AuPill>
 
-  <AuDropdown
```

Comment:
remove instead of comment

Reply:
Wasn't sure if you where a fan of the new one but apparently you are! Will remove. Should i also move the admin label to our index js file? Than this component can be removed

Reply:
Removed component